### PR TITLE
Hughsie/drm ddc

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by superm1** [comment]
Rebase against main?

File: plugins/mediatek-scaler/fu-mediatek-scaler-device.c [hunk 1]
```diff
@@ -36,100 +36,18 @@
...
-	return fu_mediatek_scaler_device_set_i2c_dev(self, i2c_devices, error);
-}
+G_DEFINE_TYPE(FuMediatekScalerDevice, fu_mediatek_scaler_device, FU_TYPE_DRM_DEVICE)
 
 static gboolean
 fu_mediatek_scaler_device_ddc_write(FuMediatekScalerDevice *self,
 				    GByteArray *st_req,
 				    GError **error)
 {
+	FuI2cDevice *ic2_proxy = FU_I2C_DEVICE(fu_device_get_proxy(FU_DEVICE(self)));
```
Comment (CragW) [inline plugins/mediatek-scaler/fu-mediatek-scaler-device.c:50]: `i2c_proxy` perhaps?

Comment (hughsie) [reply plugins/mediatek-scaler/fu-mediatek-scaler-device.c:50]: Oopa, fixed, thanks.

File: plugins/mediatek-scaler/mediatek-scaler.quirk [hunk 2]
```diff
@@ -1,8 +1,2 @@
-[PCI\VID_1028&PID_0C88]
-Flags = probe-vcp
-[PCI\VID_1028&PID_0C8A]
-Flags = probe-vcp
-[PCI\VID_1028&PID_0CE9]
-Flags = probe-vcp
-[PCI\VID_1028&PID_0CEB]
-Flags = probe-vcp
+[DRM\VEN_DEL&DEV_4340]
+Plugin = mediatek_scaler
```
Comment (hughsie) [inline plugins/mediatek-scaler/mediatek-scaler.quirk:2]: @CragW any more  needed here?

**Comment by hughsie** [comment]
@CragW do you want me to merge this, or wait until we've got the other DDC models? I'd like to include this in 2.0.0 as then I drop some then-unused udev API.

**Comment by CragW** [comment]
> @CragW do you want me to merge this, or wait until we've got the other DDC models? I'd like to include this in 2.0.0 as then I drop some then-unused udev API.

I'm in line with this approach, let's see @GregLo007 .

File: plugins/mediatek-scaler/mediatek-scaler.quirk (hunk 2 above)
Comment (CragW) [reply plugins/mediatek-scaler/mediatek-scaler.quirk:2]: Yes, we need some additions. Thanks for checking.
```
[DRM\VEN_DEL&DEV_7430]
Plugin = mediatek_scaler
```

**Review by CragW — APPROVED** [review]

**Comment by CragW** [comment]
closes #7568.

**Pull request merged**